### DISCUSSION TITLE
Save playground conversation to a dataset

### DISCUSSION THREAD

**Discussion by bderenzi** [langfuse/langfuse#2071] [Ideas]
### Describe the feature or potential improvement

The playground is a good way to explore edge cases of a prompt. Creating a trace and being able to add it to a dataset would be useful

**Comment by brendanhillovida** [comment]
We would get a lot more benefit from the Datasets feature if we could import a thread from playground to dataset.

**Comment by BryanA2T** [comment]
We run complex chats with multiple threads. Creating datasets in playground is the only viable way to di this efficiently.

**Comment by xamey** [comment]
I support this. Would love to easily add entries to datasets from traces + playgrounds

**Comment by cristopherfreitas** [comment]
I think just adding the ability to save from the playground and then loading from a dataset to a playground would be a huge win for experimentation.
